src/test/groovy/io/openliberty/tools/gradle/TestMultiModuleLooseEarWithPages.groovy

```diff
@@ -0,0 +1,120 @@
+                </archive>
+            </archive>
+            <archive targetInArchive="/ejb-jar-1.0-SNAPSHOT.jar">
+                <dir sourceOnDisk="/Users/arunkumarvn/Documents/public/ci.gradle/build/testBuilds/multi-module-loose-ear-pages-test/jar/build/classes/java/main" targetInArchive="/"/>
+                <file sourceOnDisk="/Users/arunkumarvn/Documents/public/ci.gradle/build/testBuilds/multi-module-loose-ear-pages-test/jar/build/resources/tmp/META-INF/MANIFEST.MF" targetInArchive="/META-INF/MANIFEST.MF"/>
+            </archive>
+        <file sourceOnDisk="/Users/arunkumarvn/Documents/public/ci.gradle/build/testBuilds/multi-module-loose-ear-pages-test/ear/build/tmp/META-INF/MANIFEST.MF" targetInArchive="/META-INF/MANIFEST.MF"/>
+    </archive>
+    */
+    @Test
+    public void test_loose_config_file_contents_are_correct(){
+        File on = new File("build/testBuilds/multi-module-loose-ear-pages-test/ear/build/wlp/usr/servers/ejbServer/apps/ejb-ear-1.0-SNAPSHOT.ear.xml");
+        FileInputStream input = new FileInputStream(on);
+        String ejbWar = "/ejb-war-1.0-SNAPSHOT.war"
+        String warWebappsFolder = "/multi-module-loose-ear-pages-test/war/src/main/webapp"
+        String ejbJar = "/WEB-INF/lib/ejb-jar-1.0-SNAPSHOT.jar"
+
+
+        // get input XML Document
+        DocumentBuilderFactory inputBuilderFactory = DocumentBuilderFactory.newInstance();
+        inputBuilderFactory.setIgnoringComments(true);
+        inputBuilderFactory.setCoalescing(true);
+        inputBuilderFactory.setIgnoringElementContentWhitespace(true);
+        inputBuilderFactory.setValidating(false);
+        inputBuilderFactory.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false)
+        inputBuilderFactory.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false)  
+        DocumentBuilder inputBuilder = inputBuilderFactory.newDocumentBuilder();
+        Document inputDoc=inputBuilder.parse(input);
+
+        // parse input XML Document
+        XPath xPath = XPathFactory.newInstance().newXPath();
+        String expression = "/archive/dir";
+        NodeList nodes = (NodeList) xPath.compile(expression).evaluate(inputDoc, XPathConstants.NODESET);
+        Assert.assertEquals("Number of <dir/> element ==>", 0, nodes.getLength());
+
+        expression = "/archive/archive";
+        nodes = (NodeList) xPath.compile(expression).evaluate(inputDoc, XPathConstants.NODESET);
+        Assert.assertEquals("Number of <archive/> element ==>", 2, nodes.getLength());
+        if (OSUtil.isWindows()) {
+            ejbWar = "//ejb-war-1.0-SNAPSHOT.war"
+            warWebappsFolder = "//multi-module-loose-ear-pages-test//war//src//main//webapp"
+            ejbJar = "//WEB-INF//lib//ejb-jar-1.0-SNAPSHOT.jar"
+        }
+        Assert.assertTrue(List.of(nodes.item(0).getAttributes()
```

Comment:
This is failing in the build:

<img width="1483" alt="Screenshot 2024-10-16 at 10 51 23 AM" src="https://github.com/user-attachments/assets/c0472897-9bc0-48c5-9634-946ee55ec75a">

Reply:
probably not available with Java 8